test/lib/fixtures/projects/publish.js

```diff
@@ -0,0 +1,31 @@
+var retrieveTestProjects = require('./test-projects');
+
+var published = {};
+var userToken = {
+  authorization: 'token ag-dubs'
```

Comment:
can we remove "specific people" from this codebase and instead use generic names or "testuser" or something? In part because people leave an org but more importantly because people new to the code will have no idea whether something is a username if it's a non-obvious sequence of letters that look like it might be meaningful code, such as with 'ag-dubs'